vuepress/docs/next/docs/getting-started/development-concepts.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebarDepth: 2
+---
+
+# Development Concepts on Entando
+ 
+On Entando Composition Platform, the development of an application is broken down into modular steps of create, curate, compose, and consume. In the simplest terms, you start by **creating** functional building blocks, **curate** and organize the blocks in a catalogue, **compose** them on a page or website, which is then **consumed** in an app where it can be scaled and improved upon. 
```

Comment:
"On Entando Composition Platform" -> "With the Entando Platform,"
"modular steps" -> "distinct steps" or "discrete steps" Not sure there.
"catalogue" -> "catalog".  Let's use American English here.